src/snowflake/snowpark/functions.py

```diff
@@ -5790,6 +5793,25 @@ def vector_cosine_distance(v1: ColumnOrName, v2: ColumnOrName) -> Column:
     return builtin("vector_cosine_distance")(v1, v2)
 
 
+def vector_cosine_similarity(v1: ColumnOrName, v2: ColumnOrName) -> Column:
```

Comment:
Is this a PuPr or GA function?

Reply:
It's GA (the one being deprecated was only PrPr and was deprecated before PuPr).